### PR TITLE
Regenerate client-protocol classes following unused import suppression

### PR DESCRIPTION
**Pull request by JackPGreen**
The generated client-protocol classes are full of unused imports - in https://github.com/hazelcast/hazelcast-client-protocol/issues/482 the generator was changed to add a suppression to these generated classes.

This PR is to import these regenerated classes.

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_2_1.java [hunk 1]
```diff
@@ -6600,13 +6601,11 @@ public void test_CPSubsystemAddGroupAvailabilityListenerCodec_decodeResponse() {
 
     private static class CPSubsystemAddGroupAvailabilityListenerCodecHandler extends CPSubsystemAddGroupAvailabilityListenerCodec.AbstractEventHandler {
         @Override
-        public void handleGroupAvailabilityEventEvent(com.hazelcast.cp.internal.RaftGroupId groupId, java.util.Collection<com.hazelcast.cp.CPMember> members, java.util.Collection<com.hazelcast.cp.CPMember> unavailableMembers,
-                                                      boolean isIsShutdownExists, boolean isShutdown) {
+        public void handleGroupAvailabilityEventEvent(com.hazelcast.cp.internal.RaftGroupId groupId, java.util.Collection<com.hazelcast.cp.CPMember> members, java.util.Collection<com.hazelcast.cp.CPMember> unavailableMembers, boolean isIsShutdownExists, boolean isShutdown) {
             assertTrue(isEqual(aRaftGroupId, groupId));
             assertTrue(isEqual(aListOfCpMembers, members));
             assertTrue(isEqual(aListOfCpMembers, unavailableMembers));
-            assertFalse(isIsShutdownExists); // < 2.7
```
Comment (zpqrtbnk) [inline hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_2_1.java:6608]: most changes are expected header changes, but on this file and on all other tests that have changed... is that an expected change?

**Review by gbarnett-hz — APPROVED** [review]
For the `isShutdown` related tests -- that's fine; the lines affected were additional assertions I added.

File: hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_2_1.java (hunk 1 above)
Comment (JackPGreen) [reply hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_2_1.java:6608]: > most changes are expected header changes, but on this file and on all other tests that have changed... is that an expected change?

yes and no - they were last changed in c15043352a2ec157ccf36458e767c935ecadcafb, but it looks like they were manually modified on top of being automatically generated (e.g. difference in formatting, comments etc).
Then when I've regenerated the code, those manual changes have been undone showing a difference.
It's only a change to the tests and they still pass, so I'm happy.

It'd probably be "cleaner" to regenerate the code from the previous commit to fix that, and then make this change, but I'm not sure that's worth it.

Comment (srknzl) [reply hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityTest_2_1.java:6608]: The files are not expected to be modified anyway. Let's ignore the changes.

**Review by srknzl — APPROVED** [review]

**Pull request merged**